resources/assets/components/Users/UserInformation.js

```diff
@@ -14,7 +14,20 @@ const UserInformation = props => (
         }
         <p>
           {props.user.email ? <span>{props.user.email}<br /></span> : null}
-          {props.user.mobile ? <span>{props.user.mobile}<br /></span> : null }
+
+          {
+            (() => {
+              if (props.user.mobile) {
+                if (props.linkSignup && isEmpty(props.user.first_name)) {
+                  return (<span><a href={`/signups/${props.linkSignup}`}>{props.user.mobile}</a><br /></span>);
+                }
+
```

Comment:
I truly don't know if there should be a newline here or not, but I think this and the next return on line 27 should be consistent in that regard?

Reply:
I forget if this in our code style, but I'd vote for a newline in both! 🍃

Reply:
thanks @katiecrane and @DFurnes ! A newline above the `return`? Just want to make sure I understand correctly! If above the return, should I also put a newline above the `return` in line 22?